src/components/CreatePromoter/CreatePromoter.module.scss

```diff
@@ -1,5 +1,13 @@
-@import '~@santiment-network/ui/variables';
+@import '~@santiment-network/ui/mixins';
 
 .container {
   padding: 24px;
+
+  @include responsive('phone', 'phone-xs') {
```

Comment:
Use webkit's global class for responsive rules. For example: https://github.com/santiment/app/blob/a84ca36df1bca1902c185e5c5db286bf92b43efa/src/pages/TrendsExplore/EnhancedWordCloud.module.scss#L22)

`@include responsive` can be leaved and extended in old code parts (e.g. `src/pages/Account/AccountPage.module.scss` `LOC 78`), meanwhile new changes should be written using global classes

Reply:
Resolved, Please review